src/style.css

```diff
@@ -1,3 +1,122 @@
-body {
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-style: normal;
+}
+
+.body {
+  margin: 0;
+}
+
+.card {
```

Comment:
You should create separate file for each BEM block (Checklist p11)

So create separate files for `card`, `stars` etc